app/views/layouts/application.html.erb

```diff
@@ -81,7 +82,7 @@
             Menu
           </button>
           <nav>
-            <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+            <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
```

Comment:
Admin is not "active" here:
<img width="1064" alt="Screen Shot 2019-08-27 at 20 06 29" src="https://user-images.githubusercontent.com/464306/63801131-808a1d80-c907-11e9-9b74-cd181724a192.png">

do you also need something like this? https://github.com/alphagov/govwifi-product-page/blob/555747c93081cd771869fdb0e77e5009ab0300fd/config.rb#L50-L53

Reply:
Admin actually seems to disappear and get replaced by Sign out once you log in. Not sure what the accessibility says on that (although it's not on scope to change it now)

Reply:
the active menu item functionality was done for the Product Page in the same PR as the new headers. In hindsight it was out of scope, so decided to not attempt it here. Are we happy with a new ticket to track that work?